graylog2-web-interface/src/pages/ServerUnavailablePage.jsx

```diff
@@ -112,9 +111,9 @@ class ServerUnavailablePage extends React.Component {
                 <a href="https://www.graylog.org/community-support" rel="noopener noreferrer" target="_blank">We can help you</a>.
               </p>
               <div>
-                <a role="button" tabIndex={0} onClick={this._toggleDetails}>
-                  {this.state.showDetails ? 'Less details' : 'More details'}
-                </a>
+                <Button bsStyle="link" tabIndex={0} onClick={this._toggleDetails}>
+                  {showDetails ? 'Less details' : 'More details'}
+                </Button>
```

Comment:
Can we use a default or info button here? I know it already has been a link, but when i clicked on the link I expected a redirect. This could also fix this padding "problem".

Before:
![image](https://user-images.githubusercontent.com/46300478/75542196-98aa3900-5a1f-11ea-96e7-3ecf6367bfae.png)

After:
![image](https://user-images.githubusercontent.com/46300478/75542227-a5c72800-5a1f-11ea-9aa0-60be6eec4163.png)

Reply:
![image](https://user-images.githubusercontent.com/122591/75705194-811dbb00-5c80-11ea-922d-96f3ee6b1c46.png)
![image](https://user-images.githubusercontent.com/122591/75705199-84b14200-5c80-11ea-8806-86eb3ff63984.png)